docs/usage.md

```diff
@@ -185,27 +179,100 @@ You can also supply a run name to resume a specific run: `-resume [run-name]`. U
 
 Specify the path to a specific config file (this is a core Nextflow command). See the [nf-core website documentation](https://nf-co.re/usage/configuration) for more information.
 
-#### Custom resource requests
+## Custom configuration
+
+### Resource requests
+
+Whilst the default requirements set within the pipeline will hopefully work for most people and with most input data, you may find that you want to customise the compute resources that the pipeline requests. Each step in the pipeline has a default set of requirements for number of CPUs, memory and time. For most of the steps in the pipeline, if the job exits with any of the error codes specified [here](https://github.com/nf-core/rnaseq/blob/4c27ef5610c87db00c3c5a3eed10b1d161abf575/conf/base.config#L18) it will automatically be resubmitted with higher requests (2 x original, then 3 x original). If it still fails after the third attempt then the pipeline execution is stopped.
```

Comment:
I wanted to link to an explicit commit because if the code changes in the future then we would still be pointing to exactly the same line of code.

Reply:
I completely agree with the idea, but in the end, since we only do have one specific commit for a specific tag, it would end up being the same code, and a much nicer link.

Reply:
Ah, I see what you mean now 👍🏽 The only problem there is that if I change the code between now and the release then those links will be broken and it's something that can be quite easily overlooked 😅